pkg/kube/inject/testdata/inject/opt-in-injection.iop.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+  name: example-istiocontrolplane
+spec:
+  values:  
+    global:
+      proxy:
+        autoInject: enabled    
+      podDNSSearchNamespaces:
+      - "global"
+      - "{{ valueOrDefault .DeploymentMeta.Namespace \"default\" }}.global"
+    sidecarInjectorWebhook:
+      neverInjectSelector:
+      - matchExpressions:
+        - key: sidecar.istio.io/inject
+          operator: DoesNotExist
+      enableNamespacesByDefault: true
+      objectSelector:
+        enabled: true
+        autoInject: true
```

Comment:
This config doesn't make any sense to me. I am fine adding a test of course, but I would strongly discourage anyone from doing this...